server/data/vocabularies.json

```diff
@@ -2,7 +2,7 @@
     {
         "_id": "languages",
         "display_name": "Language",
-        "type": "unmanageable",
+        "type": "manageable",
         "selection_type": "multi selection",
         "init_version": 2,
```

Comment:
Probably need to increment `init_version` so existing instances update this CV when released to the environment

Reply:
indeed, thanks